From e8a9956556f0c4934dcc90a3acddb61630a87740 Mon Sep 17 00:00:00 2001
From: HeYue <yue.he@intel.com>
Date: Wed, 15 Mar 2023 10:20:32 +0800
Subject: [PATCH] Revert "Android.mk: Filter out --build-id=md5 linker flag"

This reverts commit 554b19b616a16ad049e73d036660a271be00c572.
Vulkan failed to run with device enumerate failure on mesa23
due to this patch.

Tracked-On: OAM-106720
Signed-off-by: Fei Jiang fei.jiang@intel.com
---
 android/mesa3d_cross.mk | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/android/mesa3d_cross.mk b/android/mesa3d_cross.mk
index 727aaac32dd7..738f865836b1 100644
--- a/android/mesa3d_cross.mk
+++ b/android/mesa3d_cross.mk
@@ -169,14 +169,13 @@ define m-lld-flags
 endef
 
 define m-lld-flags-cleaned
-  $(patsubst -Wl$(comma)--build-id=%,, \
   $(subst prebuilts/,$(AOSP_ABSOLUTE_PATH)/prebuilts/, \
   $(subst $(OUT_DIR)/,$(call relative-to-absolute,$(OUT_DIR))/, \
   $(subst -Wl$(comma)--fatal-warnings,,                \
   $(subst -Wl$(comma)--no-undefined-version,,          \
   $(subst -Wl$(comma)--gc-sections,,                   \
   $(patsubst %dummy.o,,                                \
-    $(m-lld-flags))))))))
+    $(m-lld-flags)))))))
 endef
 
 define m-cpp-flags
-- 
2.17.1

